src/userSync.js

```diff
@@ -225,6 +225,13 @@ export function newUserSync(userSyncDependencies) {
       }
       return checkForFiltering[filterType](biddersToFilter, bidder);
     }
+
+    // iframes are disabled by default.
+    // so if the iframe config is invalid, we should use the permittedPixels object to tell us if the bidder should be blocked when trying to register an iframe pixel.
+    if (!isFilterConfigValid(filterConfig, type) && type === 'iframe') {
+      return !permittedPixels[type];
+    }
+
     return false;
```

Comment:
It looks to me that all these lines can be replaced with simple:

```
return !permittedPixels[type];
```

There is no need to check `!isFilterConfigValid(filterConfig, type)`, previous `if` is already checking that condition, in other words it will always evaluate to `true`.

Reply:
Makes sense. Change applied

Reply:
Build is failing, you have linting error:

```
/home/circleci/Prebid.js/src/userSync.js
  231:3  error  Block must not be padded by blank lines  padded-blocks

✖ 1 problem (1 error, 0 warnings)
  1 error, 0 warnings potentially fixable with the `--fix` option.

[14:12:48] 'lint' errored after 28 s
[14:12:48] ESLintError in plugin "gulp-eslint"
Message:
    Failed with 1 error
Details:
    domainEmitter: [object Object]
    domainThrown: false

[14:12:48] 'test' errored after 28 s
```